use-latest-deps-android.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
```

Comment:
If you wanted to, you could just write this part in Python, too.

Reply:
Yep but I wanted to just copy what the other repos were doing for consistency / smaller error surface.

Reply:
@tswast loves bash but i've long lamented that I want to re-write all of these in Python. ;)

(If only I had time)

Reply:
Lol. I wouldn't say I love it, but bash and I are good friends at this point.

Seems like [GitPython](https://gitpython.readthedocs.io/en/stable/intro.html) plus [PyGitHub](http://pygithub.readthedocs.io/en/latest/introduction.html) could replace the commit-and-push and send-pr scripts.

Reply:
You wouldn't even need to do that, you can just use `subprocess.check_call()`.